qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -0,0 +1,268 @@
+        description: "QA Pipeline: ((pipeline-name)) (SLES <%= avail %>)"
+        path: commit-id/sha
+        state: pending
+    <% end %> # status_reporting
+    on_failure:
+      put: pool.kube-hosts
+      params: {release: pool.kube-hosts}
+  - do:
+    <% if upgrade_from_version %>
+    - task: cf-deploy-pre-upgrade
+      file: ci/qa-pipelines/tasks/cf-deploy.yml
+      params:
+        KUBE_REGISTRY_HOSTNAME: ((registry-hostname))
+        KUBE_REGISTRY_USERNAME: ((registry-username))
+        KUBE_REGISTRY_PASSWORD: ((registry-password))
+        KUBE_ORGANIZATION: ((organization))
+        HA: <%= avail == 'HA' %>
+        MAGIC_DNS_SERVICE: ((magic-dns-service))
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(upgrade_from_version) %>
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% if enable_cf_usb_deploy_pre_upgrade %>
+    - task: usb-deploy-pre-upgrade
+      file: ci/qa-pipelines/tasks/usb-deploy.yml
+      params:
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+    <% end %> # enable_cf_usb_deploy_pre_upgrade
+    <% if enable_cf_smoke_tests_pre_upgrade %>
+    - task: cf-smoke-tests-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(upgrade_from_version) %>
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+        TEST_NAME: smoke-tests
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_smoke_tests_pre_upgrade
+    <% if enable_cf_brain_tests_pre_upgrade %>
+    - task: acceptance-tests-brain-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(upgrade_from_version) %>
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+        TEST_NAME: acceptance-tests-brain
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_brain_tests_pre_upgrade
+    <% if enable_cf_acceptance_tests_pre_upgrade %>
+    - task: acceptance-tests-pre-upgrade
+      file: ci/qa-pipelines/tasks/run-test.yml
+      params:
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(upgrade_from_version) %>
+        GKE_PRIVATE_KEY_BASE64: ((gke-private-key-base64))
+        TEST_NAME: acceptance-tests
+      input_mapping:
+        s3.scf-config: s3.scf-config-sles
+    <% end %> # enable_cf_acceptance_tests_pre_upgrade
+    <% if enable_cf_deploy %>
+    - task: cf-upgrade
+      file: ci/qa-pipelines/tasks/cf-upgrade.yml
+      params:
+        KUBE_REGISTRY_HOSTNAME: ((registry-hostname))
+        KUBE_REGISTRY_USERNAME: ((registry-username))
+        KUBE_REGISTRY_PASSWORD: ((registry-password))
+        KUBE_ORGANIZATION: ((organization))
+        HA: <%= avail == 'HA' %>
+        MAGIC_DNS_SERVICE: ((magic-dns-service))
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(enable_cf_deploy) %>
```

Comment:
`<%= bundle_url_from_version(enable_cf_deploy) %>` just becomes an empty string right? Why don't we just use an empty string here then?